tests/slow/src/test/scala/tests/MavenSlowSuite.scala

```diff
@@ -0,0 +1,200 @@
+package tests
+
+import scala.meta.internal.metals.Messages._
+import scala.meta.io.AbsolutePath
+import scala.meta.internal.builds.MavenDigest
+import java.nio.file.Files
+import java.nio.file.Paths
+import scala.collection.JavaConverters._
+import scala.meta.internal.metals.ServerCommands
+
+object MavenSlowSuite extends BaseImportSuite("maven-import") {
+
+  val defaultPomPath = Paths.get(
+    this.getClass().getClassLoader().getResource("test-pom.xml").toURI()
+  )
+  val defaultPom = Files
```

Comment:
FYI, we have a `AbsolutePath.readText` helper to read paths into strings. In this particular case, the idiomatic way to read the file from resources is

https://github.com/scalameta/metals/blob/f28c35ac855ed58bd5bf30fcea70e8497c49e697/metals-bench/src/main/scala/bench/SourceCompletion.scala#L44-L50

No need for `getResource("...").toURI().toPath.readText`

Reply:
Fixed